pkgs/applications/editors/jupyter-kernels/octave/kernel.nix

```diff
@@ -0,0 +1,29 @@
+{ lib, octave, python3Packages }:
+
+with python3Packages;
+
+buildPythonPackage rec {
+  pname = "octave_kernel";
```

Comment:
```suggestion
  pname = "octave-kernel";
```

Reply:
Are you sure about this? I noticed that a number of the packages in `python-packages.nix` still have underscores. It seems good to stay consistent with the Python module name, which has the underscore

Reply:
done